portfolio/src/main/webapp/script.js

```diff
@@ -48,4 +48,13 @@ function addRandomFunFact() {
   // Add it to the page.
   const funfactContainer = document.getElementById('funfact-container');
   funfactContainer.innerText = funfact;
+}
+
+/**
+ * Fetch greeting message from DataServlet and add it to the page
```

Comment:
Per https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments, you should write function declaration comments in the third person.